modules/microshift-nw-topology.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// * microshift_networking/microshift-cni.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="microshift-network-topology_{context}"]
+= Network topology
+OVN-Kubernetes provides an overlay-based networking implementation. This overlay includes an OVS-based implementation of Service and NetworkPolicy. The overlay network uses the Geneve (Generic Network Virtualization Encapsulation) tunnel protocol. The pod maximum transmission unit (MTU) for the Geneve tunnel is set to a smaller value than the MTU of the physical interface on the host. This smaller MTU makes room for the required information that is added to the tunnel header before it is transmitted.
```

Comment:
This statement is inaccurate. Please refer to https://github.com/openshift/microshift/blob/f0d95cf39d316f396cea7af4c329d8de9389f579/docs/contributor/network/default_cni_plugin.md?plain=1#L68